### PR TITLE
Use virtualised list for task table

### PR DESCRIPTION
**Pull request by juhoeerola**
Also moved the task filtering out from the task table component.
It is now the user's responsibility to select which tasks should be
shown.

### CODE REVIEW

**Review by IidaKainu — COMMENTED** [review]
I'll drop a single suggestion since I am unable to do a full review today

**Review by IidaKainu — CHANGES_REQUESTED** [review]
Excluding the previous suggestion (which is why this is submitted as 'request changes') I have not much to comment on. I learned a lot from reading the code and this is obviously well executed. 🔥

File: frontend/src/components/TaskTable.tsx [hunk 1]
```diff
@@ -88,186 +99,60 @@ const TaskTable: FC<{
...
-      label="Rated tasks"
-      TaskRow={TableRatedTaskRow}
-    />
+        ))}
+      </div>
+      <hr style={{ width: '100%' }} />
+      <FixedSizeList
+        itemSize={80}
+        itemCount={sorted.length}
+        height={600}
```
[the remark this answers is not in the document]
Comment (juhoeerola) [reply frontend/src/components/TaskTable.tsx]: Good catch, also made the height and row height parameters

**Comment by juhoeerola** [comment]
Noticed that Roadmap.users could be undefined, so added that to the type

**Review by IidaKainu — APPROVED** [review]

**Pull request merged**